components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -507,13 +507,28 @@ func (backend *ES2Backend) GetReport(esIndex string, reportId string,
 	return report, errorutils.ProcessNotFound(nil, reportId)
 }
 
+func arrayIncludes(a []string, matchString string) bool {
+	for _, itemInArray := range a {
+		if itemInArray == matchString {
+			return true
+		}
+	}
+	return false
+}
+
 func convertControl(profileControlsMap map[string]*reportingapi.Control, reportControlMin ESInSpecReportControl, filters map[string][]string) *reportingapi.Control {
 	profileControl := profileControlsMap[reportControlMin.ID]
 
 	if profileControl == nil {
 		return nil
 	}
 
+	if len(filters["control"]) > 0 {
+		if !arrayIncludes(filters["control"], profileControl.Id) {
```

Comment:
minor, but if you want, I think this is the same as:

```suggestion
		if !stringutils.SliceContains(filters["control"], profileControl.Id) {
```

which would allow you to remove lines 510-518 above.

Reply:
ah, thank you, yes!